pandas/core/nanops.py

```diff
@@ -1240,7 +1307,7 @@ def nancorr(a, b, method="pearson", min_periods=None):
     return f(a, b)
 
 
-def get_corr_func(method):
+def get_corr_func(method) -> Callable:
```

Comment:
method: Union[str, Callable]

Reply:
In https://github.com/pandas-dev/pandas/pull/30461/commits/7442310a426f9160abab5e201c10b8ad4590b25d I removed all annotations of ```str``` and ```Callable``` that were added in this PR, it just gave me too much trouble.